CMakeLists.txt

```diff
@@ -230,10 +231,13 @@ elseif(CMAKE_SYSTEM_NAME STREQUAL "Darwin")
 elseif(CMAKE_SYSTEM_NAME STREQUAL  "Android")
   add_definitions(-DOS_LINUX)
   add_definitions(-DOS_ANDROID)
+  add_definitions(-DCUSTOM_LOG)
+  add_definitions(-DENABLE_LOG)
   SET(BUILD_BT_SUPPORT OFF)
   SET(BUILD_USB_SUPPORT ON)
```

Comment:
Please use lowercase command (Change from SET to set)in CMake to keep use same command format.